src/app/pages/home/services/tasks.service.ts

```diff
@@ -100,12 +118,13 @@ export class TasksService {
             )
```

Comment:
In the above part, why is ESM also not calculated valid according to the completionWindow ?